### ISSUE TITLE
Possible issue with export presets...seem to be missing and importing them doesn't work...

### ISSUE THREAD

**Issue by todd-prior** [darktable-org/darktable#12884]
I haven't exported any files recently but I had a batch of edits today so I went to export them and my presets were all gone. I checked other modules and my presets are there.. I will go back and look at some backup db. What is funny is I have recently also exported my presets and so I had them handy. I try to import them and no error is thrown but they are not added ...

Have also tried to restart after the import and still not showing as options

I will see if this is the case for any other modules but from a quick look they seem fine..

Has anyone else noticed this??

I will provide more detail if needed...

So basically anyone running WIn and using a recent DT version.. 

Mine is   

![image](https://user-images.githubusercontent.com/44409419/201508792-190c3aa2-83d4-4435-94bb-57a9ff754d64.png)

Not sure if any work on renaming things or something else?? Or just a freak incident on my end...

Edit also I see that I cannot open the save preset dialogue in the Export module...selecting save preset does nothing but it is fine for all other modules...

**Comment by pehar1** [comment]
Seems similar to https://github.com/darktable-org/darktable/issues/12818. Should be fixed by https://github.com/darktable-org/darktable/commit/6eacb1adcc39dd052a8466ce5794023b7db53043 . You can verify this by creating a fresh build from current master. You will probably have to recreate your presets (for the export module) or to restore them from a backup of `data.db`.

**Comment by Nilvus** [comment]
@todd-prior: this issue had been fixed. @pehar1 is right. So please update to last master release (you are near 50 commits late). This should work (if not, recreate your presets). I close this issue as already fixed.

**Comment by todd-prior** [comment]
I tried with last nights build... Still wasn't able to save a new preset in the export module.  I think the issue is now the database is somehow corrupted. When I renamed my database and let DT create a blank one I can save a new preset in export module but not with the last few renditions of my database... So I guess I will have to decide to go back even farther or perhaps just export everything from my most recent database ... Let DT make a new one and then import them back...

**Comment by Nilvus** [comment]
Keep your actual database in case, your issue is always there with new database. If issue still there, provide new infos here. This could be reopen if needed.

**Comment by TurboGit** [comment]
@todd-prior : 
> Still wasn't able to save a new preset in the export module. I think the issue is now the database is somehow corrupted.

Really strange. Maybe running dt with `-d sql` will help understand the issue.

**Comment by todd-prior** [comment]
I went back a couple of more snapshots with different file sizes and got one that had pretty much everything and then ran the latest code as of noon.... seems to be fine now.... thx
